### DISCUSSION TITLE
wrong color of preview from iphone 13

### DISCUSSION THREAD

**Discussion by centralhardware** [photoprism/photoprism#2769] [Q&A]
preview 
![image](https://user-images.githubusercontent.com/26829359/194626875-db71683a-2415-4078-8395-e2045eae610d.png)
first frame
![image](https://user-images.githubusercontent.com/26829359/194626945-2760fdc8-5a74-4e72-a467-34ad7acd4d00.png)

other example:
preview
![image](https://user-images.githubusercontent.com/26829359/194627106-5ec0b456-00b7-4512-846d-0d7d14b1ab04.png)
first frame of video:
![image](https://user-images.githubusercontent.com/26829359/194627177-b008a121-ecb2-47c3-8c32-916396202bc9.png)

source files:
[first video](https://drive.google.com/file/d/1vBLPv3epSKXRZjGkErkzfEJmYlZ9Lh9_/view?usp=drivesdk)
[second video
](https://drive.google.com/file/d/1mxij96JJEIVdM5u8v0FMUZp47PPTkgqE/view?usp=drivesdk)
looks like preview generation error.

**Comment by lastzero** [comment]
I'd assume this is because the video uses the P3 Display color profile, but FFmpeg does not export the frame with icc color profile header. You could help by doing research if there's a better way to create still images from Apple iPhone videos. Maybe there's an embedded JPEG we could extract?
